validationFunctions/validateBusinessType.js

```diff
@@ -20,13 +20,13 @@ const validateBusinessType = type => {
       return false;
     }
 
-    const trimmedValidBusinessTypes = validBusinessTypes.map(validType =>
-      trim(validType)
-    );
-
-    const matchesValidBusinessTypesList = trimmedValidBusinessTypes.includes(
-      trim(type)
-    );
+    let matchesValidBusinessTypesList = false;
+    Object.keys(businessTypeEnum).forEach(validType => {
```

Comment:
is this the same as doing `return businessTypeEnum[trim(type)] ? true : false` ?

Reply:
Yes, will update to match the others enum validators